src/integrators/integrator_interface.jl

```diff
@@ -12,14 +12,17 @@ function DiffEqBase.change_t_via_interpolation!(integrator::ODEIntegrator,t,modi
     end
     integrator.t = t
     integrator.dt = integrator.t - integrator.tprev
-    reeval_internals_due_to_modification!(integrator)
+    DiffEqBase.reeval_internals_due_to_modification!(integrator)
     if T
       solution_endpoint_match_cur_integrator!(integrator)
     end
   end
 end
+DiffEqBase.change_t_via_interpolation!(integrator::ODEIntegrator,
```

Comment:
why two functions?

Reply:
We want to make sure that the first arg of `change_t_via_interpolation!` is specialized, yet, it needs to take both `ODEIntegrator` and `DDEIntegrator`.

Reply:
why not a union?

Reply:
oooooh I see. Ahh yeah, since we need something for DelayDiffEq.jl to overload. Add a comment.